strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryFiles.java

```diff
@@ -47,6 +49,20 @@ public static Boolean isArtifact(RepositoryPath path)
         return (Boolean) Files.getAttribute(path, formatAttributes(RepositoryFileAttributeType.ARTIFACT));
     }
 
+    public static boolean wasModifiedAfter(RepositoryPath path,
```

Comment:
> this method should be in RepositoryFiles

It's already there

Reply:
> also we have ArtifactEntry.lastUpdated which should be used instead of Files.getLastModifiedTime(path) if RepositoryFiles.isArtifact(repositoryPath) == true

it complicates things since we need it in `org.carlspring.strongbox.providers.layout.Maven2LayoutProvider#getRepositoryFileAttributes` to calculate `EXPIRED` `RepositoryFileAttributeType`

Reply:
I think it should work, can you please check?

Reply:
I'll try tomorrow

Reply:
`RepositoryFiles.isArtifact(repositoryPath)` returns `false` for `maven-metadata.xml`

Reply:
`repositoryPath.artifactEntry` is `null`